api/system-apps/read

```diff
@@ -46,11 +46,45 @@ sub read_json
     return decode_json($json);
 }
 
+sub invoke_info_api
+{
+    use IPC::Open2;
+    my $helper_path = shift;
+    my $helper_input = shift;
+    my($chld_out, $chld_in, $pid);
+
+    $helper_input = {%$input, 'action' => 'app-info', %$helper_input}; # merges the input hashes
+    $pid = open2($chld_out, $chld_in, $helper_path);
+    print $chld_in encode_json($helper_input);
+    close($chld_in);
+    my $raw_response = <$chld_out>;
+    close($chld_out);
+    waitpid( $pid, 0 );
+    return safe_decode_json($raw_response);
+}
+
 sub app_info
 {
     my $app = shift;
     my $file = "$path$app.json";
     my $data = read_json($file);
+    my $extended_data = {};
+
+    # If available, invoke the INFO_API provider
+    # to extend the manifest data dynamically
+    if($data->{'INFO_API'}) {
```

Comment:
I'd change the key to `infoapi` or `info_api`. Everything else is in lower case, and I'd like to respect the conventions.